content/en/workflows/actions_catalog/generic_actions.md

```diff
@@ -4,7 +4,88 @@ kind: documentation
 disable_toc: false
```

Comment:
The sidebar and table of contents appear to be missing from the preview. I wonder if it has to do with how the directory is set up for the Actions catalog? Probably good to ask Websites about it after Dash or whenever things calm down a bit.

Reply:
Thanks for pointing this out! I added a reminder to my calendar to follow up on this for the Monday following Dash